src/components/__tests__/MyCalendar.test.ts

```
@@ -119,24 +126,23 @@ describe('MyCalendar', () => {
     await wrapper.vm.$nextTick()
-    expect(wrapper.findComponent(EventCard).props('isOpen')).toBe(true)
+    // The EventCard should NOT open when clicking a day cell (only when clicking an event)
+    expect(wrapper.findComponent(EventCard).props('isOpen')).toBe(false)
+    // Optionally, check for day-selected event emission if you want:
+    // expect(wrapper.emitted('day-selected')).toBeTruthy()
   })
 
   it('goes to today when clicking today button', async () => {
     const todayBtn = wrapper.findAll('button').find(btn => btn.text() === 'Today')
     expect(todayBtn).toBeTruthy()
     await todayBtn!.trigger('click')
```

Biome (1.9.4)

[error] Forbidden non-null assertion.

Unsafe fix: Replace with optional chain operator ?. This operator includes runtime checks, so it is safer than the compile-only non-null assertion operator


(lint/style/noNonNullAssertion)